### PR TITLE
Add shadows to TD projectiles and fix Napalm bombs

### LINKED ISSUE #12799: Add projectile shadows to TD
TD has received little attention this release cycle (except for the major balancing overhaul and upcoming map pool overhaul, of course), so we should at least match RA regarding projectile shadows.

Now that weapons use inheritance, this should be fairly easy to do.

### LINKED ISSUE #12800: TD Napalm bomb hits ground instantly
Regression from #12405 (https://github.com/OpenRA/OpenRA/commit/f30f47de4f85aca8321442e1dd881dab2902715a).

### PR DESCRIPTION
**Pull request by reaperrr**
Closes #12799.
Fixes #12800.

### CODE REVIEW

File: mods/cnc/weapons/missiles.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@
 		Blockable: false
 		Inaccuracy: 128
 		Image: DRAGON
+		Shadow: true
```
Comment (pchote) [inline mods/cnc/weapons/missiles.yaml]: You can now remove the redundant `Shadow` definition from `227mm`.

**Review by pchote — APPROVED** [review]
Looks good to me otherwise.  Any chance you could squeeze a fix for #12803 in here too?

**Comment by reaperrr** [comment]
> Looks good to me otherwise. Any chance you could squeeze a fix for #12803 in here too?

Sure.

**Comment by reaperrr** [comment]
Updated.

**Review by ltem — APPROVED** [review]
- Code: no issues
- Behavior: no missing shadows found, Napalm bombs are fixed

**Comment by ltem** [comment]
:+1:

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/30d70397fce12527ee115048caa6cb3f1e390f89...cae90139b3376c1e02b73c8b2f3807cbdd55de84)

**Pull request merged**